odd_collector/domain/plugin.py

```diff
@@ -39,6 +39,7 @@ class ClickhousePlugin(DatabasePlugin):
 
 class RedshiftPlugin(DatabasePlugin):
     type: Literal["redshift"]
+    schemas: list = []
```

Comment:
Use List from typing, with explicit type of elements

Reply:
updated